src/functionalTest/resources/features/F-206 - Request Notice Of Change/F-206 - Request Notice Of Change.feature

```diff
@@ -0,0 +1,336 @@
+      And it is submitted to call the [Request NoC] operation of [Manage Case Assignment Microservice],
+
+    Then a positive response is received,
+      And the response has all the details as expected,
+      And a call [to verify that the case is not listed in Dil's organisation] will get the expected response as in [XXX],
+      And a call [to verify that Dil has not been granted any case roles for the Org.] will get the expected response as in [AAA],
+      And another call [to verify that Mario now observes a pending NOC request on the case] will get the expected response as in [YYY],
+      And another call [to verify that a caseworker in C1's jurisdiction now observes a pending NOC request on the case] will get the expected response as in [YYY].
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# ACA-68 / AC-7
+  @S-206.7
+  Scenario: (Happy Path) : Solicitor successfully Requests NoC to Replace a litigant's current representation - Auto-approval applies
+
+    Given an appropriate test context as detailed in the test data source,
+      And a user [Richard - with the ability to create a case for a particular jurisdiction within an organisation],
+      And a user [Matt - with a caseworker-pui role for a different organisation to that of Richard],
+      And a user [Dil - with a solicitor role for the same jurisdiction within a different organisation from both Richard and Matt],
+      And a successful call [by Richard to create a case - C1 which contains 3 Org Policies for 3 case roles: R1, R2 & R3 which is auto-assigned to Richard's organisation] as in [Prerequisite Case Creation Call for Case Assignment],
+      And a successful call [by Matt to trigger a NoC Request which contains answers identifying case role R1],
+
+    When a request is prepared with appropriate values,
+      And the request [is made by Dil to place a NOC Request for C1],
+      And the request [contains all correct answers in the correct format - e.g. a Date of Birth provided in Date format],
+      And the request [contains answers identifying case role R2],
+      And it is submitted to call the [Request NoC] operation of [Manage Case Assignment Microservice],
+
+    Then a positive response is received,
+      And the response has all the details as expected,
+      And a call [to verify that the case is listed in Dil's organisation] will get the expected response as in [XXX],
+      And a call [to verify that Dil has been granted case roles R1 & R2] will get the expected response as in [AAA],
+      And a call [to verify that Dil has not been granted case role R3] will get the expected response as in [BBB],
+      And another call [to verify that Richard does not observe a pending NOC request on C1] will get the expected response as in [YYY].
```

Comment:
NB: usual two verify calls suggested for AC1 would remain applicable. although feel free to clarify which roles: e.g. 
* [to verify that Dil has been granted case roles R1 & R2 for the case but not R3]